src/node/oscillator.rs

```diff
@@ -292,7 +124,7 @@ pub struct OscillatorNode {
     /// starts and stops Oscillator audio streams
     scheduler: Scheduler,
     /// channel between control and renderer parts (sender part)
-    sender: Sender<OscMsg>,
+    sender: Sender<OscillatorMessage>,
```

Comment:
Not really related to your work, but I don't see why this is not a `Sender<PeriodicWave>`

Reply:
Yup you are right, I will update that